toolset/databases/sqlserver/create.sql

```diff
@@ -0,0 +1,52 @@
+create database hello_world;
+go
+use hello_world;
+go
+create table world (
+  id int not null identity(1,1) primary key,
```

Comment:
memory optimised? (using instructions from https://docs.microsoft.com/en-us/sql/relational-databases/in-memory-oltp/demonstration-performance-improvement-of-in-memory-oltp?view=sql-server-2017)
```sql
create database hello_world;
go
alter database hello_world add filegroup hello_world_mod contains memory_optimized_data;
alter database hello_world add file
 (name = hello_world_dir, filename= '/sqlserver/hello_world_dir')
 to filegroup hello_world_mod;
go
use hello_world;
go
create table world (
  id int not null identity(1,1) primary key nonclustered hash with (bucket_count = 70000),
  randomNumber int not null
) with (memory_optimized = on, durability = schema_and_data);
go
```

Reply:
This could also be `durability = schema_only`; but then it doesn't persist changes to disk and they are lost across restarts - so that seems more niche than general purpose (e.g. cache/temp data)

Reply:
Probably should also be simple recovery mode, so:
```sql
create database hello_world;
go
alter database hello_world set recovery simple; 
go
alter database hello_world add filegroup hello_world_mod contains memory_optimized_data;
alter database hello_world add file
 (name = hello_world_dir, filename= '/sqlserver/hello_world_dir')
 to filegroup hello_world_mod;
go
use hello_world;
go
create table world (
  id int not null identity(1,1) primary key nonclustered hash with (bucket_count = 70000),
  randomNumber int not null
) with (memory_optimized = on, durability = schema_and_data);
go
```